tools/stats/import_test_stats.py

```diff
@@ -18,7 +18,7 @@ def get_disabled_issues() -> List[str]:
     return issue_numbers
 
 
-SLOW_TESTS_FILE = ".pytorch-slow-tests.json"
+SLOW_TESTS_FILE = ".pytorch-slow-tests.json?versionId=Zw9Db41MTHlq3T.gc9Si4xX8D.FAvyDC"
```

Comment:
The failure comes from this line. The `SLOW_TESTS_FILE` needs to remain the same, you have already updated the version in the URL below.

Reply:
Yup good catch.